### ISSUE TITLE
Start-DbaAzMigration

### ISSUE THREAD

**Issue by potatoqualitee** [sqlcollaborative/dbatools#6838]
No idea if this is even useful but `Start-DbaAzMigration` would be similar to `Start-DbaMigration` but it'd use bacpacs and just migrate whats supported (Databases and logins?) Also gotta test if you can create a db that doesn't exist yet.

Here's some code that I used for a single db

````
$dacpac = Export-DbaDacPackage -SqlInstance localhost -Database $dbname -Type Bacpac #-DacOption $extractOptions

$options = New-DbaDacOption -Action Publish
$results = $dacpac | Publish-DbaDacPackage -Database $dbname -SqlInstance $aserver -Confirm:$false -Type Bacpac
````

sqlpackage is available on Linux, I could make it core compat too.

**Comment by wsmelton** [comment]
Yes, bacpac can be used on new Azure SQL DBA server to create new databases. That is the primary function of it in Azure.

**Comment by potatoqualitee** [comment]
now possible with AI but i think this shouldbe in a new module possibly. table for next year when it'll come up.
